projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -624,6 +624,16 @@ export interface FeatureTogglesInterface {
    */
   a11yQTY2Quantity?: boolean;
 
+  /**
+   * In `CardComponent`, place `Delete` button before `Cancel` button.
+   */
+  a11yDeleteButton2First?: boolean;
```

Comment:
I think this has been included by accident

Reply:
I've just resolved it.